lib/taskana-core/src/main/java/pro/taskana/impl/TaskanaEngineImpl.java

```diff
@@ -356,14 +356,17 @@ protected void initRoles(String propertiesFilename, String propertiesSeparator)
 
                     InputStream inputStream = this.getClass().getResourceAsStream(propertiesFile);
                     if (inputStream == null) {
-                        LOGGER.error("properties file {} was not found on classpath.", propertiesFile);
+                        LOGGER.error("properties file {} for Role configuration was not found on classpath.",
+                            propertiesFile);
                         ensureRoleMapIsFullyInitialized();
                         return;
                     } else {
                         props.load(new InputStreamReader(inputStream));
+                        LOGGER.info("Role properties were loaded from file {} from classpath.", propertiesFile);

```

Comment:
loglvl should be DEBUG in my optinion. 
Important logs wouldn´t be unseen if a to high amount of logentries. 

If you agree with this please have a look at all loglevels on this PR.